src/Google.Cloud.Functions.Invoker/HostingExtensions.cs

```diff
@@ -50,7 +46,7 @@ public static class HostingExtensions
         /// </summary>
         /// <param name="builder">The logging builder to configure.</param>
         /// <returns>The original builder, for method chaining.</returns>
-        public static ILoggingBuilder ConfigureFunctionsFrameworkLogging(this ILoggingBuilder builder, WebHostBuilderContext context) =>
+        public static ILoggingBuilder AddFunctionsFrameworkLogging(this ILoggingBuilder builder, WebHostBuilderContext context) =>
```

Comment:
This was deliberately Configure rather than Add because it does more than add - it clears existing providers.